x-pack/plugins/fleet/server/services/epm/elasticsearch/template/template.test.ts

```diff
@@ -1105,7 +1105,12 @@ describe('EPM template', () => {
   runtime: true
 `;
     const runtimeFieldMapping = {
-      properties: {},
+      properties: {
+        labels: {
+          type: "object",
+          dynamic: true,
+        },
```

Comment:
Example of intermediary object are created as static property for dynamic templates.